__tests__/categories.test.js

```diff
@@ -0,0 +1,24 @@
+// const seed = require('../db/seeds/seed.js')
+// const data = require('../db/data/test-data/index.js')
+const request = require('supertest')
+const app = require('../app.js')
```

Comment:
We do want to be seeding our database before each test - see the notes on seeding with pg for a refresher!